client/Makefile

```diff
@@ -49,11 +44,8 @@ smoke-test:
 # start an instance of the Explorer server and run the end-to-end annotations tests
 .PHONY: smoke-test-annotations
 smoke-test-annotations:
-	$(eval TMP_DIR := $(shell  mktemp -d /tmp/cellxgene_XXXXXX))
-	cp $(ANNOTATIONS) $(TMP_DIR)/ && \
-	cp $(GENE_SETS) $(TMP_DIR)/ && \
 	start_server_and_test \
-		'CXG_OPTIONS="--annotations-file $(TMP_DIR)/$(ANNOTATIONS_FILENAME) --gene-sets-file $(TMP_DIR)/$(GENE_SETS_FILENAME)" $(MAKE) start-server' \
+		'CXG_OPTIONS="" $(MAKE) start-server' \
```

Comment:
TODO: Do these annotations smoke tests have value without these user annotation files in place? Do the tests still exercise the client side-only annotations features?  (@tihuan Maybe you know?)

Reply:
Hmm that's a great question! 

We have `npm run e2e-annotations` to test client side annotations features, but since we have the desktop repo to do that and don't have annotations for hosted Explorer, it feels like we can actually delete all annotations related smoke test in this repo?

@MDunitz @seve  does that sound right to you?

Thank you!

Reply:
So I think these [tests](https://github.com/chanzuckerberg/single-cell-explorer/blob/32d5f08381cf9f1152c1f0ffc845cb001c57dc48/client/__tests__/e2e/cellxgeneActions.ts) _are_ still valid, and should have no usage of those annot/gene set files that were being specified. But bumping @MDunitz and @seve in case you know for sure.

Reply:
That should be fine. I think removing all of the annotations code is the plan for this quarter

Reply:
@seve can look and confirm. Maybe a story to refactor this tests as separate effort

Reply:
Can verify there's value in `client/__tests__/e2e/e2eAnnotations.test.ts`that  needs to be replaced.

* "geneSET crud operations and interactions"
* "GENE crud operations and interactions"
* "stacked bar graph renders"
  * Need to look into this one, but can't remember why this was implemented as a annnotations test
 * "truncate midpoint whitespace"
    * Easily added as a new category to test dataset
 * "truncate single character"
    * Likewise

Reply:
![image](https://user-images.githubusercontent.com/8716829/150024908-d9e050aa-691b-4f1f-b0f9-95cffa507cbe.png) Feel free to push the button!

Reply:
Thx, @seve! Created https://github.com/chanzuckerberg/single-cell-explorer/issues/213.  If that looks right, you can resolve this thread.